src/styles/blocks/_calendar.scss

```diff
@@ -0,0 +1,64 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 10px;
+  margin-top: 20px;
```

Comment:
I think it's better to use padding in this case.
1. ".calendar" - this is a BEM block and you cannot set margin for them;
2. You need this place on top for full animation display.